.sass-lint.yml

```diff
@@ -1,4 +1,5 @@
 rules:
+  class-name-format: 0
```

Comment:
Instead of getting rid of the class name formatting rules, what about using the [hyphenatedbem](https://github.com/benthemonkey/sass-lint/blob/feature/class-name-format/docs/rules/class-name-format.md#example-6) formatting convention instead? I'm fine to change the standard, but I do think it makes sense to enforce some kind of consistency.